Assets/Scripts/State/IdleState.cs

```diff
@@ -26,11 +29,41 @@ public IdleState(Character character, State nextState = null)
 
         public override void Update(float deltaTime)
         {
+                Tile startTile = character.CurrTile;
+                Tile endTile = null;
+                List<Tile> path = null;
+
+                // Get a random Int 0,1,2,3 for representing the 4 directions.
+                int direction = Int32.Parse(Math.Truncate(Random.value * 4).ToString());
+
+                switch (direction)
+                {
+                    case 0:
+                        endTile = World.Current.GetTileAt(startTile.X + 1, startTile.Y, startTile.Z);
+                        break;
+                    case 1:
+                        endTile = World.Current.GetTileAt(startTile.X, startTile.Y + 1, startTile.Z);
+                        break;
+                    case 2:
+                        endTile = World.Current.GetTileAt(startTile.X - 1, startTile.Y, startTile.Z);
+                        break;
+                    case 3:
+                        endTile = World.Current.GetTileAt(startTile.X, startTile.Y - 1, startTile.Z);
+                        break;
+                }
+
```

Comment:
I personally like the first one, but they both the same really in readability so yeh choose whatever one you want really.